### ISSUE TITLE
controls in android

### ISSUE THREAD

**Issue by muvv** [TheWidlarzGroup/react-native-video#332]
Can i get example of implementing controls in android
i added controls={true} prop ,but i am not getting control options in android. It could be better if i can get a example in android

**Comment by pedramsaleh** [comment]
I don't think that's how controls work. Check the example and you'll see how to make some of the controls come up. Basically you have to implement them yourself.

**Comment by SHRoberts91** [comment]
My controls aren't showing either, and when I actually go into my node_modules, the changes that were implemented in #270 were not brought in when I npm installed,  
for example, my node_modules doesn't have  
`@ReactProp(name = PROP_CONTROLS, defaultBoolean = false)
    public void setControls(final ReactVideoView videoView, final boolean controls) {
        videoView.setControls(controls);
    }`
in this file, `android/src/main/java/com/brentvatne/react/ReactVideoViewManager.java`, and it's the same for the other changes in that commit. And so obviously `controls={true}` isn't working for me in android. 
I realized this is because the commit with these android changes(#270) hasn't been put on the stable branch yet, so if you just npm installed the stable branch you won't have access to the android controller features yet.
